conformance/lang/expressions/trap_expr.balt

```diff
@@ -0,0 +1,252 @@
+Test-Case: output
+Description: Test trap-expr when the expression is a nil-literal.
```

Comment:
In the description let's use 
```suggestion
Description: Test trap expression when the expression is a nil-literal.
```

Reply:
Please fix wherever applicable for all the productions that have been used.

Reply:
Changed